src/pages/ZNS/ZNS.tsx

```diff
@@ -103,6 +104,7 @@ const ZNS: React.FC<ZNSProps> = ({ domain, version }) => {
 
 	// Force to go back to home if invalid domain
 	React.useEffect(() => {
+		if(loading === false){
```

Comment:
Prefer to use `if (!loading)` instead of `if (loading === false)` since it's a slightly safer operation (there is a subtle different between the two)